src/prepare_dlc_dev_environment.py

```diff
@@ -33,19 +33,9 @@
 
 DEFAULT_TOML_URL = "https://raw.githubusercontent.com/aws/deep-learning-containers/master/dlc_developer_config.toml"
 
+DLC_REPO_URL = "https://raw.githubusercontent.com/aws/deep-learning-containers/master"
 
-def restore_default_toml(toml_path):
-    """
-    Restore the TOML file to its default state from the specified URL
-    """
-    try:
-        response = requests.get(DEFAULT_TOML_URL)
-        response.raise_for_status()
-        with open(toml_path, "w") as toml_file:
-            toml_file.write(response.text)
-        LOGGER.info(f"Restored {toml_path} to its default state from {DEFAULT_TOML_URL}")
-    except requests.exceptions.RequestException as e:
-        LOGGER.error(f"Error restoring {toml_path}: {e}")
+TEMP_BUILD_FILE = ".prepare_dlc_files_to_revert.txt"
```

Comment:
Provide a comment about what this is - you're storing buildspec files that can be reverted